tools/splitfasta/splitFasta.xml

```diff
@@ -1,17 +1,32 @@
-<tool id="rbc_splitfasta" name="Split Fasta" version="0.2.0">
+<tool id="rbc_splitfasta" name="Split Fasta" version="0.3.0">
     <description>files into a collection</description>
     <requirements>
         <requirement type="package" version="1.65">biopython</requirement>
     </requirements>
-    <stdio>
-        <exit_code range="1:" />
-    </stdio>
-    <command interpreter="python">
+    <stdout>
+        <exit_code range="1:" level="fatal" />
+    </stdout>
+    <command interpreter="python" detect_errors="aggressive">
```

Comment:
You should remove the deprecated interpreter tag and use `python $__tool_directory__/split_fasta.py` instead.
Moreover, if you use detect_erroros you can omit the stdout section.